lib/cmd/common/import.js

```diff
@@ -93,7 +93,17 @@ function importAppNgui(projectId, appTitle, appTemplateType, fileOrRepo, deployE
       'User-Agent' : "FHC/" + fhc._version + ' ' + os.platform() + '/' + os.release()
     };
 
-    var r = request.post({url: url, headers:headers}, function(err, res, data) {
+    var formData = {
+      title: appTitle,
+      templateType: appTemplateType,
+      templateZipFile: fs.createReadStream(file)
+    };
+
+    if (deployEnv) {
+      formData.autoDeployOption = deployEnv;
+    }
+
+    var r = request.post({url: url, headers:headers, formData: formData}, function(err, res, data) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'r' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AUnusedVariable)